src/components/ai-input.tsx

```diff
@@ -136,16 +145,15 @@ export const AIInput = ( {
 					<>
 						<MenuGroup>
 							<MenuItem
+								isDestructive
```

Comment:
Leveraging this built-in prop over custom styles ensures a red focus border is displayed and increases app UI consistency.